### DISCUSSION TITLE
Task manager says Laragon is 32bit + fails to load randomly

### DISCUSSION THREAD

**Discussion by dontletthesamedogbiteyoutwice** [leokhoa/laragon#396] [General]
This morning Laragon failed to start. Checking Task manage shows it's 32 bit. Thought it was a 64 bit app.

Ending the task in Task manger and trying again, most times, Laragon will start.

Any ideas what might be causing  this?
![laratask](https://user-images.githubusercontent.com/112656038/216840001-8686b036-880c-4466-a1a3-f08c4c9a282c.png)
![notresponing](https://user-images.githubusercontent.com/112656038/216840002-c6be15c3-2e8b-4588-adc2-62fad7f129fd.png)

**Comment by Pen-y-Fan** [comment]
My task manager shows **Laragon (32 bit)** too. I guess Laragon will launch 32-bit or 64-bit apps, so being 32-bit gives greater portability.

![Laragon (32-bit)](https://user-images.githubusercontent.com/40126936/216843127-b5ad533f-9b1d-4d34-9207-4cd3877c6dcf.png)

Anti-virus app would be my first suspect for something preventing Laragon from launching. Check if you can configure an exclusion for Laragon.exe or the laragon folder.

See my blog [How to set up Laragon on a new Windows computer (part 1) - Install Laragon](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/), part 2 has details on permissions, it maybe worth applying permission for your user account.

**Comment by dontletthesamedogbiteyoutwice** [comment]
//edit
not sure what's happening. Will post back what I find.

I discovered that if I have a Browser, Chrome, FF, etc. open, Laragon won't start.

With no browsers open it starts fine (!?)

**Comment by dontletthesamedogbiteyoutwice** [comment]
I tried installing php 7.x and mysql 5.x - no change

The work around for me was to NOT have Laragon "start all automatically". Now any php or mysql version works without issue. Just use Start ALL.

I've backed up my DB's and may uninstall and start anew.
